React/Base/RCTConvert.m

```diff
@@ -567,6 +571,25 @@ static BOOL RCTFontIsCondensed(UIFont *font)
   return (symbolicTraits & UIFontDescriptorTraitCondensed) != 0;
 }
 
+NSDictionary *RCTFontVariantMapping = NULL;
+
++ (NSDictionary *)getRCTFontVariantMapping
+{
+  if (!RCTFontVariantMapping) {
+    RCTFontVariantMapping = @{
+      @"tabular-nums": @{
```

Comment:
It looks like these options are mutually-exclusive? How does this work when you can provide an array of them?